packages/bpk-component-scrollable-calendar/src/utils.ts

```diff
@@ -31,16 +31,16 @@ const getMonthsArray = (startDate, count) => {
 // Here we calculate the height of each calendar grid item in pixels, as the `react-window` API
 // requires that these are provided so that they can be efficiently rendered.
 const getMonthItemHeights = (
-  months,
-  weekStartsOn,
-  columnCount,
-  rowHeight,
-  baseMonthItemHeight,
-) => {
+  months: Date[],
+  weekStartsOn: 0 | 1 | 2 | 3 | 4 | 5 | 6,
+  columnCount: number,
+  rowHeight: number,
+  baseMonthItemHeight: number,
+): number[] => {
   const monthItemHeights = months.map((month) => {
     const firstDayOffset = (month.getDay() + 7 - weekStartsOn) % 7;
     const monthLength = DateUtils.daysInMonth(
-      month.getYear(),
```

Comment:
this is deprecated, but using `getFullYear` will give the same results in this case